client/src/components/UserAvatarName.vue

```diff
@@ -5,10 +5,12 @@
     <ion-avatar
       class="avatar person-avatar body-sm"
     >
-      {{ user.substring(0, 2) }}
+      {{ userAvatar.substring(0, 2) }}
     </ion-avatar>
-    <ion-text class="person-name">
-      {{ user }}
+    <ion-text
```

Comment:
Should probably add a `v-show` on the `ion-text` so that the tag is not displayed if userName is not set.

Reply:
Yes true, I will be show only if userName is `true`